### PR TITLE
Ra filters

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Moved from #26692

[Internal review URL](

### CODE REVIEW

File: aspnetcore/fundamentals/minimal-apis/min-api-filters.md [hunk 1]
```diff
@@ -82,13 +82,13 @@ Consider a filter that validates a `Todo` object:
 In the preceding code:
 
 * The `RouteHandlerInvocationContext` object provides access to the [`MethodInfo`](/dotnet/api/system.reflection.methodinfo) associated with the endpoint's handler and the `EndpointMetadata` that has been applied on the endpoint.
```
Comment (WeihanLi) [inline aspnetcore/fundamentals/minimal-apis/min-api-filters.md:84]:
```suggestion
* The `EndpointFilterInvocationContext` object provides access to the [`MethodInfo`](/dotnet/api/system.reflection.methodinfo) associated with the endpoint's handler and the `EndpointMetadata` that has been applied on the endpoint.
```

Forgot to update this, my bad

**Comment by Rick-Anderson** [comment]
@WeihanLi your updates will be live in 15 minutes along with the code updates. Can you review the updated tutorial and make sure it's OK?

**Comment by WeihanLi** [comment]
@Rick-Anderson looks great, thanks very much

**Comment by Rick-Anderson** [comment]
@WeihanLi  do you want to PR that?

**Pull request merged**